includes/controls/groups/flex-container.php

```diff
@@ -202,16 +202,34 @@ protected function init_fields() {
 
 		$fields['align_content'] = [
 			'label' => esc_html__( 'Align Content', 'elementor' ),
-			'type' => Controls_Manager::SELECT,
+			'type' => Controls_Manager::CHOOSE,
+			'label_block' => true,
 			'default' => '',
 			'options' => [
-				'' => esc_html__( 'Default', 'elementor' ),
-				'center' => esc_html__( 'Center', 'elementor' ),
-				'flex-start' => esc_html__( 'Start', 'elementor' ),
-				'flex-end' => esc_html__( 'End', 'elementor' ),
-				'space-between' => esc_html__( 'Space Between', 'elementor' ),
-				'space-around' => esc_html__( 'Space Around', 'elementor' ),
-				'space-evenly' => esc_html__( 'Space Evenly', 'elementor' ),
+				'start' => [
```

Comment:
@rami-elementor What about BC? You changed the option keys from `flex-start` to `start` for example. The control will be empty when the page is edited again?

Reply:
Fixed. Thank you!